delete-user-data/extension.yaml

```diff
@@ -99,6 +102,20 @@ params:
       For example, if you have the collections `users` and `admins`, and each collection
       has documents with User ID as document IDs, then you can enter `users/{UID},admins/{UID}`.
 
+  - param: FIRESTORE_DELETE_MODE
+    type: select
+    label: Cloud Firestore delete mode
+    description: >-
+      How would you like Firestore to delete documents? Set to `shallow` to delete only parent documents
+      and ignore documents in subcollections.
+    options:
+      - label: Recursive
+        value: recursive
+      - label: Shallow
+        value: shallow
+    default: recursive
```

Comment:
Chatted to @laurenzlong a bit further, can we change the default to be shallow so that this isn't a breaking change? This way the functionality of the extension doesn't automatically change when the user updates, in case the user just goes with defaults when updating.

Reply:
No problem, updated it.